docs/ref/modules/management/README.md

```diff
@@ -1 +1,91 @@
-# Management API
+# Management REST API
+
+## Introduction
+
+The REST API allows the remote management of the Wazuh infrastructure such as:
+ - CRUD operations for agents
+ - CRUD operations for groups
+ - Agent's assignments to groups
+ - CRUD operations for security entities(Users, Roles, Rules and Policies)
+ - Restarting the node(s) or agent(s)
+
+ ## Configuration
+
+| Option | Description | Mandatory | Default |
+|--------|-------------|-----------|---------|
+| host | The host address for the communications API |  | localhost |
```

Comment:
The default value has been modified. Please check https://github.com/wazuh/wazuh/blob/main/etc/wazuh-server.yml.